sdk/program/src/lib.rs

```diff
@@ -544,6 +544,8 @@ pub mod address_lookup_table_account {
     pub use crate::address_lookup_table::AddressLookupTableAccount;
 }
 
+#[deprecated(since = "2.1.0", note = "Use `solana-sanitize` crate instead")]
```

Comment:
```suggestion
#[deprecated(since = "2.0.1", note = "Use `solana-sanitize` crate instead")]
```

Reply:
Are you planning to have this backported @buffalojoec ? If not, v2.1.0 is correct.

Reply:
No - wasn't planning to backport. Good catch, apologies @kevinheavey.